newIDE/app/src/UI/SortableVirtualizedItemList/ItemRow.js

```diff
@@ -28,7 +28,7 @@ type Props = {
   selected: true,
   onItemSelected: () => void,
   buildMenuTemplate: Item => any,
-  hasErrors: boolean,
+  errorStatus: 0 | 1 | 2,
```

Comment:
Here I think it would be better to have `errorsStatus: '' | 'error' | 'warning`.

Reply:
I was thinking the same thing, but wasnt sure. Will do the changes tonight :)